packages/now-cli/src/util/projects/get-projects-with-domains.ts

```diff
@@ -0,0 +1,39 @@
+import Client from '../client';
+import wait from '../output/wait';
+import { Project } from '../../types';
+import { URLSearchParams } from 'url';
+
+export async function getProjectsWithDomains(
+  client: Client
+): Promise<Project[] | Error> {
+  const cancelWait = wait(`Fetching projects with domains`);
+  try {
+    const limit = 50;
+    let result: Project[] = [];
+
+    const query = new URLSearchParams({
+      hasProductionDomains: '1',
+      limit: limit.toString(),
+    });
+
+    for (;;) {
```

Comment:
```suggestion
    for (let i = 0; i < 1000; i++) {
```

Reply:
Yea, I think we can do that.
If the CLI should ever list more, we should provide an option to continue at a specific point.